### PR TITLE
Add mailjet/mailjet-bundle recipe

### PR DESCRIPTION
**Pull request by lbassin**
| Q             | A
| ------------- | ---
| License       | MIT

Related : https://github.com/mailjet/mailjetBundle/issues/7

### CODE REVIEW

File: mailjet/mailjet-bundle/1.0/config/packages/mailjet.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+mailjet:
+    api_key:    "%env(MJ_APIKEY_PUBLIC)%"
```
Comment (OskarStark) [inline mailjet/mailjet-bundle/1.0/config/packages/mailjet.yaml]: could you change `MJ` to `MAILJET`?

Comment (Nyholm) [reply mailjet/mailjet-bundle/1.0/config/packages/mailjet.yaml]: Yeah, It is better to be explicit.

Comment (lbassin) [reply mailjet/mailjet-bundle/1.0/config/packages/mailjet.yaml]: You're right! It's fixed :)

**Comment by lbassin** [comment]
Hi @Nyholm, are the changes good for you ?

**Pull request merged**